spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/QueryableStoreRegistry.java

```diff
@@ -44,9 +46,14 @@
 	public <T> T getQueryableStoreType(String storeName, QueryableStoreType<T> storeType) {
 
 		for (KafkaStreams kafkaStream : kafkaStreams) {
-			T store = kafkaStream.store(storeName, storeType);
-			if (store != null) {
-				return store;
+			try{
+				T store = kafkaStream.store(storeName, storeType);
```

Comment:
Sorry for being a nit pick, but I thing it's pretty safe to just return here:
```
return kafkaStream.store(storeName, storeType);
```
I mean no need for the following IF statement. . . kind of redundant.

@sobychacko if you're ok with it please make modification as part of the commit polishing before merging

Reply:
Chatted with @olegz and we need to preserve the existing behavior.